pkgs/development/compilers/chicken/5/overrides.nix

```diff
@@ -1,2 +1,51 @@
-{
+{ pkgs, lib, chickenEggs }:
+let
+  addToBuildInputs = pkg: old: {
+    buildInputs = (old.buildInputs or [ ])
+      ++ (if lib.isList pkg then pkg else [ pkg ]);
+  };
+  addToBuildInputsWithPkgConfig = pkg: old:
+    (addToBuildInputs pkg old) // {
+      nativeBuildInputs = (old.nativeBuildInputs or [ ]) ++ [ pkgs.pkg-config ];
+    };
```

Comment:
Can we break this up to `pkg: old: addPkgConfig (addToPropagatedBuildInputs pkg old)`?

Reply:
done